src/run_solver.jl

```diff
@@ -28,6 +28,7 @@ benchmark (default: `[:name, :nvar, :ncon, :status, :elapsed_time, :objective, :
 """
 function solve_problems(
```

Comment:
This works, but you have to update the unit tests (in the folder `test`) so that each call to `solve_problems` has now 3 arguments as input.

Reply:
Oh yeah... Sorry for that. This is really specific to this package.
Among the things that SolverBenchmark do, it contains a comparison of versions, and this fails is there are modified files (because it try to checkout with git if I remember correctly). So, yes you have to commit.

You can push once it works, and if you want to keep your history clean then you can "git squash"